packages/shield-controller/src/backend.ts

```diff
@@ -236,6 +266,30 @@ export class ShieldRemoteBackend implements ShieldBackend {
       Authorization: `Bearer ${accessToken}`,
     };
   }
+
+  #cancelPreviousPendingRequests(id: string) {
```

Comment:
```suggestion
  #cancelPendingRequests(id: string) {
```
If it's "pending" request, it's always a "previous" request. So we can omit `previous`.

Reply:
All these logics were refactored and removed here, https://github.com/MetaMask/core/pull/6847/commits/565c42c45d82d8021565d57a3c6dc99994795665